src/Common/src/Interop/Unix/System.Security.Cryptography.Native/Interop.Ssl.cs

```diff
@@ -133,6 +134,25 @@ internal static SafeSharedX509NameStackHandle SslGetClientCAList(SafeSslHandle s
             return handle;
         }
 
+        internal static bool AddExtraChainCertificates(SafeSslHandle sslContext, X509Chain chain)
+        {
+            Debug.Assert(chain != null, "X509Chain should not be null");
+            
```

Comment:
Debug.Assert(chain.ChainElements.Count > 0, "chain.Build should have already been called");